WordPress/WordPressShareExtension/ShareViewController.swift

```diff
@@ -59,22 +66,23 @@ class ShareViewController: SLComposeServiceViewController {
     }
     
     override func didSelectPost() {
-        RequestRouter.bearerToken = oauth2Token! as String
+        RequestRouter.bearerToken = oauth2Token!
```

Comment:
I am beginning to dislike explicit unwrapping even though I used it a lot. Lets add a guard around this and get rid of the `!`.